sed/core/processor.py

```diff
@@ -1198,6 +1298,12 @@ def add_jitter(self, cols: Sequence[str] = None):
             cols=cols,
             cols_jittered=cols,
         )
+        if self._timed_dataframe is not None:
```

Comment:
This does not work when jittering on columns which are not in the timed dataframe.
With HEXTOF, jittering is done exclusively on dld parameters, which are all by electron and therefore do not enter thie timed_dataframe. I would suggest either removing this or setting a check for only applying jitter on columns present in the timed_dataframe.

Reply:
I implemented checks and exceptions for all these cases. Maybe one could extend the processor tests to be run for both mpes and flash test data, to include these exceptions in the tests. Could also be an issue, I think it works well.